config.js

```diff
@@ -359,20 +347,27 @@ function processAllowedGroupExport(argName, argv) {
     } 
 }
 
-function testExportsWarnings() {
-    console.log("TESTING WARNINGS:\n-------------------------------------------------------");
-    let isWarning = false;
-
-    for (const r_u of ["ranked", "unranked"]) {
-        // avoid infinite games
-        // TODO: whenever --maxpausetime gets implemented, remove this
-        if (!exports.nopause && !exports[`nopause${r_u}`]) {
-            isWarning = true;
-            console.log(`    Warning: No --nopause nor --nopause${r_u}, `
-                        + `${r_u} games are likely to last forever`); 
-        }
-    }
+function exportAllowedGroupsUnderscored(argv) {
+    processBoardsizesExport("boardsizes", argv);
+    processBoardsizesExport("boardsizesranked", argv);
+    processBoardsizesExport("boardsizesunranked", argv);
+
+    processKomisExport("komis", argv);
+    processKomisExport("komisranked", argv);
+    processKomisExport("komisunranked", argv);
```

Comment:
We probably want to change this in a speparate pr, but...

It's strange to call a function called `processKomisExport` with `("komis` every time. This could be simply

```
    processBoardsizesExport("", argv);
    processBoardsizesExport("ranked", argv);
    processBoardsizesExport("unranked", argv);

    processKomisExport("", argv);
    processKomisExport("ranked", argv);
    processKomisExport("unranked", argv);
```

Reply:
yes, but this makes processRankExport and processAllowedGroupExport easier to read as they all share the same input parameter type (full optionName +/- rankedunranked)

it may be fine to keep it as it is, do i change it according to this comment?

Reply:
I'm not talking about `processRankExport` and `processAllowedGroupExport`, those functions are just fine. I'm talking about `processBoardsizesExport` and `processKomisExport`.

Reply:
yes these should be changed, but as this pr is already crowded we may instead do it in another pr in the future as you suggested